decidim/decidim
# Fix HTML titles in admin panel (part 2)

#### :tophat: What? Why?
As Admin editing modules and their components in Decidim, when navigating between these modules and their components on these pages, the admin should see the HTML title on every page he/she navigates on.

This PR is the second part of fixing these Admin titles taking into account the components of the modules.

#### :pushpin: Related Issues
*Link your PR to an issue*
- Related to #10976 
- Fixes #7905 

#### Testing

1. Login as an admin and head over to edit.
2. Head down to a module with various components like Votings.
3. Click on a component like Accountability. 
4. Notice in HTML title on page is correctly indicating the Admin which page they're using.

### :camera: Screenshots

<img width="1433" alt="image" src="https://github.com/decidim/decidim/assets/101816158/f8244906-22d5-450e-8a84-13f43353b45e">


:hearts: Thank you!
